packages/neutrino-preset-jest/src/index.js

```diff
@@ -65,6 +65,14 @@ module.exports = neutrino => {
       };
     });
 
+  neutrino.config.module
+    .rule('lint')
+    .loader('eslint', ({ options }) => ({
```

Comment:
I know I've established the pattern elsewhere, but I'm currently changing it. Could you move the merge here to the top level?

```js
.loader('eslint', props => merge(props, {
```